test/Struct.rb

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env ruby -w
+
+require 'rmagick'
+require 'test/unit'
+require 'test/unit/ui/console/testrunner' unless RUBY_VERSION[/^1\.9|^2/]
```

Comment:
I am wondering if we still need this version check? This project only builds ruby 2.3+

Reply:
Just copy & paste. Later, we can remove all them :p

Reply:
👍